src/redis-cli.c

```diff
@@ -3029,6 +3029,7 @@ static void usage(int err) {
 "                     If this argument is used, '-a' and " REDIS_CLI_AUTH_ENV "\n"
 "                     environment variable will be ignored.\n"
 "  -u <uri>           Server URI.\n"
+"                     For connection strings without a username, use default:password@host:port. \n"
```

Comment:
Thanks, but I think we should describe the structure of the URI here, not only the default password. :)

`redis://user:password@host:port/dbnum` -- it's described on the page https://redis.io/docs/connect/cli/

And I think `rediss` is used for TLS? And dbnum is optional, user:password too I think...

Reply:
@zuiderkwast Thank you for the review. 
I have added more information about the URI. Please check it...